### DISCUSSION TITLE
Is this legit? Virus warnings after installing chia 1.1.4

### DISCUSSION THREAD

**Discussion by jsburch83** [Chia-Network/chia-blockchain#4439] [Support]
Is anyone seeing this? 

https://www.reddit.com/r/chia/comments/n90s9q/virus_warnings_after_installing_chia_114/

**Comment by lvcivs** [comment]
It was reported yesterday that someone changed the link to the Windows installer on the wiki to a "dubious" download. The correct link was restored quickly by the team, but if you happened to download the bad installer exactly at that time you should delete it and download again.

**Reply by ruigaspar** [reply]
And I suggest that you create another private key just in case...

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
